lib/surface/view.ex

```diff
@@ -0,0 +1,105 @@
+defmodule Surface.View do
+  @moduledoc """
+  Use this module on regular Phoenix views to enable .sface templates
+
+  The template files need to end with `.html.sface`
```

Comment:
Is it be possible to optionally use just `.sface`?

I mean, Surface can only be used with extended HTML (Surface) files so something like using `.json.sface` or anything similar shouldn't exist.

Reply:
Yes, adjusted it: https://github.com/surface-ui/surface/pull/543/commits/f0ccd5928199a154182a7392d95ea6e254f25a0d